modules/nf-core/fastme/meta.yml

```diff
@@ -0,0 +1,48 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "fastme"
+description: "Distance-based phylogeny with FastME"
+keywords:
+  - phylogenetics
+  - newick
+  - minimum evolution
```

Comment:
I think keywords should have no spaces, could just be replaced with an underscore.